### ISSUE TITLE
Use Carton to manage dependencies

### ISSUE THREAD

**Issue by ibnesayeed** [machawk1/docker-warrick#1]
A better way to manage dependency would be to use [Carton](http://search.cpan.org/~miyagawa/Carton-v1.0.12/lib/Carton.pm) instead of installing each dependency one by one in the Dockerfile.

**Comment by machawk1** [comment]
I have never used this. Does Carton need to first be cpan'd in? Are the dependencies akin to packages.json in node?

**Comment by ibnesayeed** [comment]
Yes, it is like packages.json in Node, requirments.txt in python or Gemfile/Bundler in Ruby.
